app/controllers/authentications_controller.rb

```diff
@@ -1,4 +1,6 @@
 class AuthenticationsController < ApplicationController
+  include UsersHelper
```

Comment:
do we need to add the UsersHelper here?

Reply:
@tansaku I saw your comment after doing changes. I will revert it back to move the method to a concern.